index.html

```diff
@@ -0,0 +1,241 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=Edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script src="https://kit.fontawesome.com/3b9cd09213.js" crossorgin="anonymous"></script>
+    <link rel="preconnect" href="https:// fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link href="https://fonts.googleapis.com/css2?family=poppins:wght@500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css" <link rel="stylesheet"
+        href="https://fonts.googleleapis.com/css2?family=Material+Symbols+Rounded:ops,wght">
+    <title>MyPorfolio</title>
+</head>
+
+<body>
+    <div class="wraper">
+        <header>
```

Comment:
- [ ] [OPTIONAL] Please use `section` tag instead of `div` tag. The div element has no special meaning. It is often used as a block of children elements. The section element introduced in HTML5 standard is used to group related elements, such as a subsection of a long article. In short, the section element provides more semantic syntax than the div element.